pepper-apis/src/test/java/org/broadinstitute/ddp/util/ActivityInstanceUtilTest.java

```diff
@@ -0,0 +1,74 @@
+package org.broadinstitute.ddp.db.dao;
+
+import java.util.concurrent.TimeUnit;
+
+import org.broadinstitute.ddp.TxnAwareBaseTest;
+import org.broadinstitute.ddp.db.TransactionWrapper;
+import org.broadinstitute.ddp.db.dto.ActivityInstanceDto;
+import org.broadinstitute.ddp.model.activity.definition.FormActivityDef;
+import org.broadinstitute.ddp.util.ActivityInstanceUtil;
+import org.broadinstitute.ddp.util.TestDataSetupUtil;
+
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ActivityInstanceUtilTest extends TxnAwareBaseTest {
```

Comment:
I found that we don't have a dedicated test checking the `ActivityInstanceUtil.isReadonly()`. We check it indirectly in the routes' tests, but I believe it's still worthwhile to add one.